### PR TITLE
Changes pgTruncate to Truncate all schemas

### PR DESCRIPTION
**Pull request by paganotoni**
As discussed in Slack if you have multiple schemas TruncateAll is only truncating tables in `public` schema, this change truncates the rest of the schema tables as well.

### CODE REVIEW

File: postgresql.go [hunk 1]
```diff
@@ -202,9 +202,9 @@ DECLARE
    _sch text;
 BEGIN
    FOR _sch, _tbl IN
-      SELECT schemaname, tablename
+      SELECT schemaname, tablename, tableowner
       FROM   pg_tables
-      WHERE    schemaname = 'public'
+      WHERE  schemaname <> 'pg_catalog' AND tableowner = current_user
```
Comment (paganotoni) [inline postgresql.go]: @stanislas-m added it here.

Comment (stanislas-m) [reply postgresql.go]: :+1:

**Comment by paganotoni** [comment]
@stanislas-m added schema_migration to the where, we don't want to have migrations table cleared, do we ?

**Comment by paganotoni** [comment]
Good catch @stanislas-m! what else do we need on this one ?

**Comment by stanislas-m** [comment]
@paganotoni We'll need to patch the other drivers, but this one is good. :+1:

**Comment by paganotoni** [comment]
Thanks @stanislas-m !

**Pull request merged**